pom.xml

```diff
@@ -16,8 +23,6 @@
     <!-- DEPENDENCIES VERSION -->
     <jdk.version>21</jdk.version>
     <cds.services.version>3.10.1</cds.services.version>
-    <spring.boot.version>3.4.0</spring.boot.version>
-    <cdsdk.version>^8</cdsdk.version>
```

Comment:
Not usages of this property?

Reply:
No, this is a leftover of the cleanup done here: https://github.com/SAP-samples/cap-sflight/pull/1455